pyproject.toml

```diff
@@ -36,10 +34,13 @@ dev = [
     "mypy",
     "ruff",
     "pytest",
-    "types-pyyaml",
     "pip-tools",
 ]
 
+consistency_with_pynxtools = [
+    "pynxtools>=0.1.1",
```

Comment:
Why did you remove pynxtools from the dependencies? If you want to run the dataconverter with STM as a standalone tool, you will need to install pynxtools during the pynxtools-stm install, too, right?

Reply:
I had the plan to discuss it today. I think ultimately all the plugins should go in that direction.

Reply:
Yeah, makes sense also with the Discord discussion 👍